docs/source/project_documentation/user_guide/project_structure.md

```diff
@@ -9,6 +9,39 @@ src/service_one/service_one_misc_definitions.yaml
 src/service_two/service_two.yaml
 ...
 ```
+src
+├── system
+│   └── system.yaml
+├── external
+|   ├── external_actors.yaml
+│   └── external_interface_messages.yaml
+├── service_one
+|   ├── service_one.yaml
+│   └── service_one_misc.yaml
+└── service_two
+    └── service_two.yaml
+
+Because AaC does not require your models to be located within the same directory or even within the same directory as your AaC package install, you can house your model files however is necessary and logical for your project needs. An example of this might look like:
+```
+system/system.yaml
+external/external_interface_messages.yaml
+external/external_actors.yaml
+service_one/service_one.yaml
+service_one/service_one_misc_definitions.yaml
+service_two/service_two.yaml
+...
+```
+
+system
+  └── system.yaml
+external
+  ├── external_actors.yaml
+  └── external_interface_messages.yaml
+service_one
+  ├── service_one.yaml
+  └── service_one_misc.yaml
+service_two
+  └── service_two.yaml
```

Comment:
Similar to my first comment, I suggest we present the structure as follows so as to avoid confusion:

```suggestion
system
  └── system.yaml
external
  ├── external_actors.yaml
  └── external_interface_messages.yaml
service_one
  ├── service_one.yaml
  └── service_one_misc.yaml
service_two
  └── service_two.yaml```
```

Note the suggestion block format requires me to put the three backticks at the end of a line instead of on a line of its own but it really should be on its own line.